test/test_helper.rb

```diff
@@ -1,10 +0,0 @@
-ENV['RAILS_ENV'] ||= 'test'
-require_relative '../config/environment'
-require 'rails/test_help'
-
-class ActiveSupport::TestCase
```

Comment:
why did you delete this file?

Reply:
I didn't delete any of those codes. I added Rubocop and set up the database.